tests/snapshots/run_file__nested_str.hvm.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/run_file/nested_str.hvm
+---
+Invalid readback: [InvalidStrTerm, InvalidStrTerm, InvalidStrTerm, InvalidStrTerm, InvalidStrTerm]
```

Comment:
The invalid readback message is very hard to understand (not due to this PR, i know)

Reply:
https://app.shortcut.com/hocomp/story/401/improve